manifests/iscsi-auth-demo-target.yaml.in

```diff
@@ -0,0 +1,158 @@
+apiVersion: v1
```

Comment:
Please remove all PV and PVC entries, they are not needed and should make this file much smaller.

Reply:
done!